when.html

```diff
@@ -0,0 +1,744 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>On-Call Load Analysis - AEM Edge Delivery Status</title>
+    <meta name="description" content="Heatmap showing likelihood of getting paged by hour and day of week">
```

Comment:
The description sounds like the audiencd is the ops team. But this is our site to communicate system status to the outside world, so it should read more like this. Let's also prevent indexing.
```suggestion
    <meta name="description" content="Heatmap showing likelihood of incidents occurring by hour and day of week">
    <meta name="robots" content="noindex, nofollow">
```